feature-configs/5g-ref-nw/README.md

```diff
@@ -0,0 +1,140 @@
+N1 interface is transparent to RAN (encapsulated in N2).
+
+Radio access network is divided to Centralized units (CU), distributed units (DU) and Radio units (RU). Centralized units are split to control plane and user plane functions (CU-CP and cu-up respectively).
+RAN entities and internal interfaces from the telecommunications standard perspective are shown below:
+
+<img src="images/ran.png" width=800>
+
+From the high level blocks composing RAN (CU, DU, RU), only CU and DU can be virtualized and implemented as cloud-native functions.
+CU / DU split is driven mostly by latency and traffic volume optimizations. 
+A DU can be seen as an application performing intensive digital signal processing, data transfer and algorithmic tasks in constrained time budgets. DU implementations often use hardware acceleration (FPGA, GPU, eASIC) for DSP processing offload. There are also software-only implementations (FlexRAN), based on AVX-512 instructions. 
+
+# Building blocks
+## Deployment types
+### Data centers
+One of the desired effects of RAN transition to the Cloud is the ability to centralize its functions in data centers, utilizing standard IT infrastructure to provide pools of centralized RAN functions. 
+A data center deployment will typically contain one or more Kubernetes clusters hosting various types of mobile network workloads:
+- A pool of RAN CU-CP and / or cu-up units
+- A pool of DU
+- Functions related to 5G core network
+- Management functions
+It is important to emphasize that any of the functions above are optional and totally depend on the mobile network operator preferences and requirements. The servers running the above workloads are assumed to have two CPU sockets.
+
+<img src="images/data_center.png" width=400>
+
+
+### Single-node clusters
+Single node clusters are deployed at the far edge in cases when there is a need to deploy a small number of cells that can be catered by a single server. This deployment type can be realized as either remote worker node of an existing cluster, or a single node cluster (e.g. Single Node Openshift). In both cases the deployment is assumed to contain a minimum of one single-socket server. From the workload perspective only RAN DUs seem to be relevant for this deployment type at the moment.
+
+
+### Three-node clusters
+Three-node clusters are deployments where minimum three physical servers are used as both infrastructure nodes and worker nodes. Additional worker nodes can be configured if needed.
+This type of deployment is expected to run on the Far Edge in cases where single node deployment is not sufficient. The workloads can vary from a small DU pool, to a full RAN and core network setup, oriented to local MNO data centers or private network deployments.
+
+
+
+## Assumptions on RAN CNF requirements
+### <a name="cu_up_nw"></a>cu-up networks
+
+<img src="images/cu-up.png" width=800>
```

Comment:
can you include an index with all the abbrevation used?
up?
nw?
nnp?

naming rules, how are they constructed?
it's almost impossible to understand for example what `sriov-nnp-cu-up-numa1-mh-u` means
I'll try my best guess here
sriov network-node-polocy central-unit user-plane(?) numa1 mid-haul u(?)

Reply:
Yes indeed. 
Will add the naming anyway :)

Reply:
Added a section to the main README